### PR TITLE
6pm 1 delete tutor assignment

### PR DESCRIPTION
**Pull request by omarbuzdar**
Added fully functioning delete button for tutor assignments and tests pass.

### CODE REVIEW

File: javascript/src/main/pages/TutorAssignment/TutorAssignments.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { fetchWithToken } from "main/utils/fetch";
 import { useAuth0 } from "@auth0/auth0-react";
```
Comment (kenny-wang6) [inline javascript/src/main/pages/TutorAssignment/TutorAssignments.js:5]: think there is a merge conflict here, not sure how to resolve, maybe professor or a TA can help

Comment (kaihilbourne) [reply javascript/src/main/pages/TutorAssignment/TutorAssignments.js:5]: I believe I fixed the merge conflict. It passed all tests and worked when I ran locally

Comment (kaihilbourne) [reply javascript/src/main/pages/TutorAssignment/TutorAssignments.js:5]: I am actually confused on how to resolve the merge conflicts. It seems that it should just replace the line that is being replaced, but github is saying it is a merge conflict still

Comment (kaihilbourne) [reply javascript/src/main/pages/TutorAssignment/TutorAssignments.js:5]: Nevermind I was just confused about how github works, i was able to resolve the conflict

**Comment by kaihilbourne** [comment]
I figured out how to use github's conflict resolver thing, and chose the correct way to merge this branch with main.

**Review by PureDreamerGk — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM... pending testing on qa.  Good for 20 points

**Pull request merged**